cypress/integration/orders/orders.PoLineSearch.spec.js

```diff
@@ -0,0 +1,73 @@
+import NewOrder from '../../support/fragments/orders/newOrder';
+import TestType from '../../support/dictionary/testTypes';
+import Orders from '../../support/fragments/orders/orders';
+import TopMenu from '../../support/fragments/topMenu';
+import OrdersHelper from '../../support/fragments/orders/ordersHelper';
+import basicOrderLine from '../../support/fragments/orders/basicOrderLine';
+
+describe('orders: Test Po line search', () => {
+  const order = { ...NewOrder.defaultOrder };
+  const orderLine = { ...basicOrderLine.specialOrderLine };
+  let orderLineNumber;
+  const searchers = [
+    { nameOfSearch: 'Keyword', valueOfLine: orderLine.titleOrPackage },
+    { nameOfSearch: 'Contributor', valueOfLine: orderLine.contributors[0].contributor },
+    { nameOfSearch: 'Requester', valueOfLine: orderLine.requester },
+    { nameOfSearch: 'Title or package name', valueOfLine: orderLine.titleOrPackage },
+    { nameOfSearch: 'Publisher', valueOfLine: orderLine.publisher },
+    { nameOfSearch: 'Vendor account', valueOfLine: orderLine.vendorDetail.vendorAccount },
+    { nameOfSearch: 'Vendor reference number', valueOfLine: orderLine.vendorDetail.referenceNumbers[0].refNumber },
+    { nameOfSearch: 'Donor', valueOfLine: orderLine.donor },
+    { nameOfSearch: 'Selector', valueOfLine: orderLine.selector },
+    { nameOfSearch: 'Volumes', valueOfLine: orderLine.physical.volumes },
+    { nameOfSearch: 'Product ID', valueOfLine: orderLine.details.productIds[0].productId },
+    { nameOfSearch: 'Product ID ISBN', valueOfLine: orderLine.details.productIds[0].productId },
+  ];
+
+  before(() => {
+    cy.getToken(Cypress.env('diku_login'), Cypress.env('diku_password'));
```

Comment:
it can be applicable to create second order with different parameters in searchable fields. In this case you will get ability to check filtering fully.

Reply:
should be fixed/discussed